go.mod

```diff
@@ -3,18 +3,18 @@ module github.com/cheqd/cheqd-node
 go 1.22.6
 
 require (
-	cosmossdk.io/api v0.3.1
-	cosmossdk.io/core v0.5.1
+	cosmossdk.io/api v0.7.6
+	cosmossdk.io/core v0.11.0
 	cosmossdk.io/errors v1.0.1
```

Comment:
I believe we also need

```
cosmossdk.io/store v1.1.1
cosmossdk.io/x/evidence v0.1.1
cosmossdk.io/x/feegrant v0.1.1
cosmossdk.io/x/upgrade v0.1.4
```

Reply:
Thanks for the reference. Yes, I'm currently working on it, which is why the PR is still in draft.